manager/examples/activated_probes/Makefile

```diff
@@ -0,0 +1,27 @@
+all: build-ebpf build run
+
+build-ebpf:
+	mkdir -p ebpf/bin
+	clang -D__KERNEL__ -D__ASM_SYSREG_H \
+		-Wno-unused-value \
+		-Wno-pointer-sign \
+		-Wno-compare-distinct-pointer-types \
+		-Wunused \
+		-Wall \
+		-Werror \
+		-I/lib/modules/$$(uname -r)/build/include \
```

Comment:
nit: you could probably extract `$(uname -r)` into a variable.

Reply:
Will do so while updating all the examples 👍